extensions/wikia/Captcha/Captcha.setup.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+/**
+ * @file
+ * @ingroup Extensions
+ */
+
+if ( !defined( 'MEDIAWIKI' ) ) {
+	exit;
+}
+
+$wgExtensionCredits['captcha'][] = [
+	'path' => __FILE__,
+	'name' => 'Captcha',
+	'author' => ['Garth Webb'],
+	'descriptionmsg' => 'captcha-desc',
+];
+
+/**
+ * The 'skipcaptcha' permission key can be given out to
+ * let known-good users perform triggering actions without
+ * having to go through the captcha.
+ *
+ * By default, sysops and registered bot accounts will be
+ * able to skip, while others have to go through it.
+ */
+$wgGroupPermissions['*'            ]['skipcaptcha'] = false;
+$wgGroupPermissions['user'         ]['skipcaptcha'] = false;
```

Comment:
Also, since we're setting '*' to false, do we need to explicitly set 'user' and 'skipcaptcha' to false also? Won't those be covered by the wildcard?